src/stylesheets/application.scss

```diff
@@ -1,9 +1,19 @@
+// Framework
 @import 'vendor/bootstrap';
-
+// functions
+@import 'functions/sizing';
+// Variables
 @import 'variables';
-
-//@import 'mixins/*';
-//@import 'base/*';
-//@import 'layouts/*';
-//@import 'components/*';
-//@import 'screens/*';
+// Base
+@import 'base/layout';
+@import 'base/fonts';
+@import 'base/typography';
+// Mixins
+@import 'mixins/auth-container';
+// Components
+@import 'components/background-layout';
+@import 'components/form-button';
+@import 'components/Auth/logo-auth';
+@import 'components/Auth/form-input-group';
```

Comment:
1. Prefer keeping all components at the same level
2. Pascal casing is reserved for JS, in SCSS use kebab-case

Reply:
Changed in [ee79344](https://github.com/rafayet-monon/surveyor/pull/12/commits/ee7934449512d5244ef6997462e00df123d58a17)